mysite/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,53 @@
+{% extends "ct/portal.html" %}
```

Comment:
@alexbojko our new templates should extend something new but not portal template.